src/edu/csus/ecs/pc2/core/Utilities.java

```diff
@@ -159,11 +161,25 @@ public static String getSecretDataPath(String baseCDPPath, String problemShortNa
     }
 
     /**
-     * Return CCS path for input data and answer file names.
+     * Return CLICS path for sample data and answer file names.
```

Comment:
I believe the above comment about what the following method is doing is wrong.  It (appears to) claim that the method is returning the CLICS path to the **_sample_** directory -- but the name (and function) of the method is `get**Secret**DataPath()`.  Please correct the comment (or indicate why the comment is correct, and fix the method name).

Reply:
Is this a bug that you have been able to reproduce?

Reply:
> Is this a bug that you have been able to reproduce?

I don't understand the above question.  The original comment is not about a "bug", it's about incorrect/illogical code.  The comment on this method says that it returns the "CLICS path for **_sample_** data and and answer files names".  The name of the method is `getSecretDataPath()`.   Either the comment is wrong, or the method is incorrectly named.  What is there to "reproduce"?